build.sbt

```diff
@@ -72,26 +72,14 @@ lazy val bigdataModules = Seq[sbt.ClasspathDep[sbt.ProjectReference]](
 lazy val allModules =
   baseModules ++ dbModules ++ jasyncModules ++ codegenModules ++ bigdataModules ++ docsModules
 
-lazy val scala213Modules =
-  baseModules ++ dbModules ++ codegenModules ++ Seq[sbt.ClasspathDep[sbt.ProjectReference]](
-    `quill-cassandra`,
-    `quill-cassandra-alpakka`,
-    `quill-cassandra-monix`,
-    `quill-cassandra-zio`,
-    `quill-orientdb`,
-    `quill-jasync`,
-    `quill-jasync-postgres`,
-    `quill-jasync-mysql`,
-    `quill-jasync-zio`,
-    `quill-jasync-zio-postgres`,
-    `quill-spark`
```

Comment:
FYI, this list was exactly corresponding to `jasyncModules ++ bigdataModules` 🤷‍♂️